linux/step01_debian9_pg_deps.sh

```diff
@@ -2,8 +2,17 @@
 
 PGVER=${PGVER:-pg96}
 
-# Postgres installation: version installed will be 9.6
-#start-recommended
-apt-get -y install postgresql
-#end-recommended
-service postgresql start
\ No newline at end of file
+if [ "$PGVER" = "pg94" ]; then
+    # Postgres installation: version installed will be 9.6
```

Comment:
Just to check: we install 9.6 if the PGVER is pg94?

Reply:
In the doc we show how to install 9.6 yes
Travis by default test with the minimal version i.e. 9.4
I can change that so that various versions are tested

Reply:
Whatever best makes sense to you is good with me here.

Reply:
(I don't think there's much value in testing more than a couple of versions, an old and a new.)